### ISSUE TITLE
Fail to open the Secreit.py file

### ISSUE THREAD

**Issue by minhtnguyenn** [SanoKyohei/Secreit#3]
Hello,

I installed all the required modules to run the Secreit.py file on Anaconda cmd. However, when I tried to run the Secreit.py file, it didn't work. I also tried to run the code one by one, but it always stopped at the first line: from keras import models. Would it be possible for you to give an instruction on how to run the Secreit.py? 

Thank you so much in advance!

**Comment by SanoKyohei** [comment]
It may be the problem of keras version. Please check it.

**Comment by minhtnguyenn** [comment]
Thank you for your reply. I checked the keras version and re-installed the exact version you required. The code ran for a bit and stopped at this error:

"ImportError: cannot import name 'Iterable' from 'collections' (C:\Users\ASUS\anaconda3\lib\collections\__init__.py)"

How could I fix this error?

**Comment by SanoKyohei** [comment]
Maybe a problem of python version

**Comment by minhtnguyenn** [comment]
Yes, it was a problem of python version. I changed the python version to 3.6 but now the code stopped at the first line after giving this long message:
Using TensorFlow backend.
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:528: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:529: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:530: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\ASUS\anaconda3\envs\Test\lib\site-packages\tensorflow\python\framework\dtypes.py:535: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
 
Could it be a problem of python again?

**Comment by SanoKyohei** [comment]
I think this is a just warning, and the code is working well. 
You can use ChatGPT for dealing with the error, which is very efficient.
